pxr/base/arch/fileSystem.h

```diff
@@ -41,9 +41,9 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 
-#if defined(ARCH_OS_LINUX)
+#if defined(ARCH_OS_LINUX) || defined(ARCH_OS_FREEBSD)
 #include <unistd.h>
-#include <sys/statfs.h>
+//#include <sys/statfs.h>
```

Comment:
If this line is unnecessary on Linux and FreeBSD, please remove it instead of commenting it out.